index.bs

```diff
@@ -62,6 +62,15 @@ Metadata Order: This version, !*, *
   "status": "Enquiry",
   "publisher": "ISO/IEC",
   "isoNumber": "ISO/IEC DIS 23000-22"
+    },
```

Comment:
(We should deal with this issue in a separate pull request.) Should we remove "DIS" from line 64 `"isoNumber": "ISO/IEC DIS 23000-22"` ?